content/source/docs/cloud/api/workspaces.html.md

```diff
@@ -174,59 +174,101 @@ _Without a VCS repository_
 ```json
 {
   "data": {
-    "id": "ws-YnyXLq9fy38afEeb",
```

Comment:
I think I would keep this order of this top level since it’s easier to follow what the data actually is. It makes it easier to understand it. I don’t think this is a blocker for merging, though, and if sorting makes this content easier to maintain then that’s fine too. 👍🏻

Reply:
Yeah, that's a fair point.  Okay i'm going to merge this in now so that the responses are correct (even if they're sorted in an unexpected way) and then I'll make a note to maybe come back and work on this some more.